README.md

```diff
@@ -121,6 +126,37 @@ kubectl rook-ceph ceph versions
 }
 ```
 
+### Debug Mode
+
+Debug Mode will be needed when cluster mon's or osd are been into error state or in the state when 
```

Comment:
For lines 131-136:
```suggestion
Debug mode can be useful when a mon or OSD needs advanced maintenance operations that require the daemon to be stopped. Ceph tools such as `ceph-objectstore-tool`,`ceph-bluestore-tool`, or `ceph-monstore-tool` are commonly used in these scenarios. Debug mode will set up the mon or OSD so that these commands can be run.

Debug mode will automate the following:
1. Scale down the existing mon or OSD deployment
2. Start a new debug deployment where operations can be performed directly against the mon or OSD without that daemon running
   a. The main container sleeps so you can connect and run the ceph commands
   b. Liveness and startup probes are removed
```